app/views/items/index.html.haml

```diff
@@ -12,10 +12,12 @@
             %table.table.table-bordered
               %tbody
                 %tr
-                  %th.tbl-header-width-90 Name
-                  %th Price
-                  %th Actions
-                = render @items
+                  %th Sr.#
+                  %th.tbl-header-width-90.tbl-header-center Name
+                  %th.tbl-header-center Price
+                  %th.tbl-header-center Actions
+                - @items.each_with_index do |item, index|
```

Comment:
I would suggest to use `render` collection instead. Checkout the example here how you can get index without actually passing it explicitly.
https://coderwall.com/p/t0no0g/render-partial-with-collection-has-hidden-index